webapp/app/model/eligibility_data.py

```diff
@@ -494,6 +497,13 @@ def elster_registration_method_must_not_be_none(cls, v):
             raise ValueError
         return v
 
+    @validator('elster_abrufcode_eligibility')
+    def elster_abrufcode_must_not_be_none(cls, v):
+        # in case of none we do not direct to the success page
+        if v == 'none':
+            raise ValueError
+        return v
```

Comment:
Having had a look at the tests, I am not sure if that is correct. Shouldn't it be:
```suggestion
    def elster_abrufcode_must_not_be_none(cls, v):
        # in case of none we do not direct to the success page
        if v != 'no':
            raise ValueError
        return v
```